components/addRecipe.js

```diff
@@ -0,0 +1,56 @@
+var Component = require('choo/component')
+var html = require('nanohtml')
+
+
+module.exports = class AddRecipe extends Component {
+  constructor (name, state, emit) {
+    super(name)
+    this.state = state
+    this.emit = emit
+    this.state = state
+    this.test = this.test.bind(this)
+    // emit(state.events.ADD_RECIPE)
+    // emit(state.events.ADD_RECIPE, recipe)
+    
+}
+
+
+
+
+  update () {
+    //   const update = this.emit !== this.state.emit
+    //   this.emit = emit
+    //   return update
+  }
+
+
+  createElement () {
+      return html`
+    <div class="col-12">
+	    <form action="/" method="POST">
+        <input type="text" name="title" id="title" /><br>
+            <textarea cols="50" rows="3" name="ingredients" id="ingredients"></textarea>
+            <textarea cols="50" rows="3" name="directions" id="directions"></textarea>
+            <input type="submit" class="btn btn-primary" onclick=${this.test}/>
```

Comment:
Remove `type="submit"`, this makes your browser literally submit the whole form on a classic HTTP request and reloads the entire page. This might be the problem actually.